app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/CommonTriageQuestionsController.scala

```diff
@@ -451,15 +482,15 @@ class CommonTriageQuestionsController @Inject() (
             _ =>
               if (isSelfUserType)
                 routes.CommonTriageQuestionsController
-                  .whoIsIndividualRepresenting()
+                  .furtherReturnHelp()
```

Comment:
these back links should be conditional as well based on whether if it's a first return or not

Reply:
I think this is right, as it's a "self user type" we only do the first return check on a representees.

Reply:
if it's not a further return, we don't want to link back to the further return help page - that page should only be shown on a further return journey

Reply:
if it's a further return for a representative, this is the wrong back link. The logic could look something like:
```scala
if(!isSelfUserType) representee.routes.RepresenteeController.checkYourAnswers()
else if(isFurtherReturn) routes.CommonTriageQuestionsController.furtherReturnHelp()
else routes.CommonTriageQuestionsController.whoIsIndividualRepresenting()	                  
```

Reply:
this is still broken for representatives on a further return